src/RequestHandler.h

```diff
@@ -74,5 +74,10 @@ class RequestHandler {
       const std::shared_ptr<Subscriber<Payload>>& subscriber) noexcept = 0;
   virtual void onSubscriberResumed(
       const std::shared_ptr<Subscriber<Payload>>& subscriber) noexcept = 0;
+
+  // TODO (T17774014): Move to separate interface
```

Comment:
Hey Alexander!  I talked with Ondrej about this.  He wanted to hold off building out the separate interface until we were ready to actually start using these calls.  For now, this PR is to demonstrate that the calls are being made when they should be.